release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1001,6 +1001,90 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+// TODO: This known issue should be removed when RHCOS images begin using RHEL 9. RHEL 9 prevents this issue from occurring.
+* Intermittently, an IBM Cloud VPC cluster might fail to install because some worker machines do not start. Rather, these worker machines remain in the `Provisioned` phase. There is a workaround for this issue. From the host where you performed the initial installation, delete the failed machines and run the installation program again. (link:https://issues.redhat.com/browse/OCPBUGS-1327[*OCPBUGS#1327*])
++
+. Verify that the status of the internal application load balancer (ALB) for the master API server is `active`.
+.. Identify the cluster's infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get infrastructure/cluster -ojson | jq -r '.status.infrastructureName'
+----
+.. Verify that the internal ALB status is `active` by running the following command:
++
```

Comment:
get the cluster region  
$oc get machines -n openshift-machine-api
...
set the target  to the cluster 
$ibmcloud target -g <cluster_ID>  -r <cluster_region>

Reply:
@MayXuQQ I'd prefer to simply your suggestion and state: "Before executing the following command, ensure you are logged into the IBM Cloud account for your cluster and target the correct region for your cluster." I think it's reasonable for the user to know how to log into ibmcloud and target the correct region (no need to give actual commands).

@mjpytlak Please insert the above sentence as follows:

_Before executing the following command, ensure you are logged into the IBM Cloud account for your cluster and target the correct region for your cluster. Verify that the internal ALB status is active by running the following command:_